src/utils/dayViewLayout.js

```diff
@@ -345,8 +345,11 @@ export function getStyledAvailabilities ({
   let styledAvailabilities = [];
   if (!unsortedAvailabilities) return styledAvailabilities;
 
-  let availabilities = unsortedAvailabilities.sort((a, b) =>
-    new Date(get(a, availabilityStartAccessor)) - new Date(get(b, availabilityStartAccessor)) || new Date(get(a, availabilityEndAccessor)) - new Date(get(a, availabilityEndAccessor))
+  let availabilities = unsortedAvailabilities
+    .slice()
+    .sort((a, b) =>
+      new Date(get(a, availabilityStartAccessor)) - new Date(get(b, availabilityStartAccessor)) ||
+      new Date(get(a, availabilityEndAccessor)) - new Date(get(a, availabilityEndAccessor))
```

Comment:
Missed this the first time: the part after the `||` will always result in zero because it's subtracting `a`'s end time from `a`'s end time, and doesn't use `b`. I think the last `new Date` is supposed to be using `b`?

Reply:
OMG, great catch! Yes, the last `new Date` should definitely be using `b`.